# The default SignatureBindingVerifier does not provide an INFO string; update
# the InstanceSessionBinder to match.
#
# It's likely that we'll want a non-empty INFO string in the future, but this
# fix requires fewer Oak changes.
--- a/oak_sdk/containers/orchestrator_crypto_client.rs
+++ b/oak_sdk/containers/orchestrator_crypto_client.rs
@@ -25,7 +25,7 @@
 };
 use oak_session::session_binding::SessionBinder;
 
-const SESSION_BINDER_INFO_STRING: &str = "oak-session-binding-key";
+const SESSION_BINDER_INFO_STRING: &str = "";
 
 struct OrchestratorCryptoClient {
     inner: GrpcOrchestratorCryptoClient<tonic::transport::channel::Channel>,
